src/content/docs/en/vs-workspace-setup.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "VS Code Workspace Setup"
+description: "This is a guide about creation and configuration of workspaces in VS Code"
+image: { src: /images/workspace-image.svg }
+---
+
+## What is it?
+According to [VS Code Docs](https://code.visualstudio.com/docs/editor/workspaces), 'workspace' is the collection of one or more folders that are opened in a VS Code window (instance).
```

Comment:
Change to:

Is a collection of one or more......

At the end of the sentence, add: If you want to learn more about it, check the vs code docs (link to it)